portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -61,16 +66,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   /**
-  *@return the max number of comments parameter of the request or DEFAULT_MAX_NUM_COMMENTS if invalid. 
+  *@return the request parameter or default value if invalid. 
   */
-  private int getMaxNumOfComments(HttpServletRequest request) {
-    String maxNumCommentsString = request.getParameter("max");
-    int maxNumComments;
+  private int getParameter(HttpServletRequest request, String parameterName, int defaultValue) {
+    String parameterString = request.getParameter(parameterName);
+    int parameter;
     try {
-      maxNumComments = Integer.parseInt(maxNumCommentsString);
+      parameter = Integer.parseInt(parameterString);
     } catch (NumberFormatException e) {
-      return DEFAULT_MAX_NUM_COMMENTS;
+      return defaultValue;
     }
-    return maxNumComments;
+    return parameter;
+  }
+
+  private Comment entityToCommentConverter(Entity entity) {
```

Comment:
We could just use the same method in Comment class.

Reply:
yes, sorry this code hasn't been merged with changes I made for last code review yet.

Reply:
merged properly now